package.json

```diff
@@ -101,6 +101,7 @@
     "lodash.takewhile": "^4.6.0",
     "lodash.uniq": "^4.5.0",
     "megalog": "^0.1.0",
+    "mockdate": "2.0.2",
```

Comment:
I've added this as a dep to fix the date to ensure the tests aren't flaky. We can probably write our own JS for this though if people object to the dep (or alternatively, let me know if other recommendations).

Reply:
We have previously [mocked the global `Date` object](https://github.com/guardian/frontend/blob/f84702f9515bc53fa58672dab90a7bb7183a5a7f/static/src/javascripts/lib/time-utils.spec.js#L6-L12) using Jest. Has this approach caused you problems?